exercises/isogram/test/Tests.hs

```diff
@@ -0,0 +1,64 @@
+{-# OPTIONS_GHC -fno-warn-type-defaults #-}
```

Comment:
given that #144 tells us this line helps for exercises with numbers but this is not one, I think it can be removed?